src/lib/drive.ts

```diff
@@ -24,155 +26,50 @@ class Drive {
-						this.filesPostHash = postContent.hash;
-						if (itemContent.content.files.length > 0) {
-							itemContent.content.files.map((file: IPCFile) => {
-								this.files.push(file);
-								return true;
-							});
-						}
-						return true;
-					}
-					return false;
-				});
-
-				if (this.filesPostHash === '') {
-					console.log('Create Post Message');
-					const newPostPublishResponse = await post.Publish({
-						APIServer: DEFAULT_API_V2,
-						channel: ALEPH_CHANNEL,
-						inlineRequested: true,
-						storageEngine: ItemType.ipfs,
-						account: this.account,
-						postType: '',
-						content: {
-							header: 'InterPlanetaryCloud2.0 - Files',
-							files: this.files,
-						},
-					});
-					this.filesPostHash = newPostPublishResponse.item_hash;
-				}
-				return { success: true, message: 'Drive loaded' };
-			}
-			return { success: false, message: 'Failed to load account' };
-		} catch (err) {
-			console.log(err);
-			return { success: false, message: 'Failed to load drive' };
-		}
-	}
-
 	public async loadShared(contacts: IPCContact[]): Promise<ResponseType> {
 		try {
 			if (this.account) {
-				console.log(contacts);
 				await Promise.all(
 					contacts.map(async (contact) => {
-						console.log('!!');
-						if (this.account!.address !== contact.address) {
-							console.log('Address contact: ', contact.address);
-							const files: IPCFile[] = [];
-							const userData = await post.Get({
-								APIServer: DEFAULT_API_V2,
-								types: '',
-								pagination: 200,
-								page: 1,
-								refs: [],
-								addresses: [contact.address],
-								tags: [],
-								hashes: [],
-							});
-
-							console.log(userData.posts);
-							await Promise.all(
-								userData.posts.map(async (postContent) => {
-									const itemContent = JSON.parse(postContent.item_content);
-
-									console.log('h', itemContent.content.header);
-									const temp = async () => {
-										if (itemContent.content.header === 'InterPlanetaryCloud2.0 - Files') {
-											console.log('Post files founded');
-											itemContent.content.files.map((file: IPCFile) => {
-												files.push(file);
-												console.log('Files: ', files);
+						const userData = await post.Get({
+							APIServer: DEFAULT_API_V2,
+							types: '',
+							pagination: 200,
+							page: 1,
+							refs: [],
+							addresses: [contact.address],
+							tags: [],
+							hashes: [],
+						});
+
+						await Promise.all(
+							userData.posts.map(async (postContent) => {
+								const itemContent = JSON.parse(postContent.item_content);
+
+								if (itemContent.content.header === 'InterPlanetaryCloud2.0 - Contacts') {
+									console.log('Post contacts founded');
+									await Promise.all(
+										itemContent.content.contacts.map(async (contactToFind: IPCContact) => {
+											if (contactToFind.address === this.account!.address) {
+												if (contact.address === this.account?.address)
```

Comment:
this.account?.address => this.account!.address, because the account it necessarily defined